apps/dsp-app/src/app/system/projects/projects-list/projects-list.component.ts

```diff
@@ -161,22 +161,30 @@ export class ProjectsListComponent implements OnInit, OnDestroy {
   }
 
   deactivateProject(id: string) {
-    this._projectApiService.delete(id).pipe(
-      tap(() => {
-        this.refreshParent.emit(); // TODO Soft or Hard refresh ?
-      })
-    );
+    this._projectApiService
+      .delete(id)
+      .pipe(
+        take(1),
```

Comment:
you can remove take(1)

Reply:
![unnamed](https://github.com/dasch-swiss/dsp-das/assets/12176105/620690f7-38cc-4371-9056-0809e2a71e83)

Reply:
Oh, nice. Done.